praetorian_cli/ui/hunt/app.py

```
@@ -0,0 +1,551 @@
+            hunts = _extract_hunts(resp)
+            if not hunts:
+                log = self.query_one("#activity-log", RichLog)
+                log.write(f"[red]Hunt {self.hunt_uuid} not found.[/red]")
+                return
+            h = hunts[0]
+            self.hunt_status = h.get('status', '')
+            self._update_context_bar()
+            self._update_status_bar(h)
+            self._emit_changes(h)
+        except Exception:
+            pass
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)